numpy/lib/tests/test__iotools.py

```diff
@@ -182,7 +182,7 @@ def test_upgrade(self):
             res = converter.upgrade(s)
             assert_(type(res) is str)
             assert_equal(res, 'a')
-            assert_equal(converter._status, 5 + status_offset)
+            assert_equal(converter._status, 8 + status_offset)
```

Comment:
The preceding comment should be updated as well I think.

Reply:
good catch! done.